signer/src/transaction_coordinator.rs

```diff
@@ -1835,15 +1821,11 @@ where
     // Ideally the code should be formulated in a way to guarantee
     // it being infallible without relying on sequentially coupling
     // expressions. However, that is left for future work.
-    fn is_coordinator(
-        &self,
-        bitcoin_chain_tip: &model::BitcoinBlockHash,
-        signer_public_keys: &BTreeSet<PublicKey>,
-    ) -> bool {
+    fn is_coordinator(&self, bitcoin_chain_tip: &model::BitcoinBlockHash) -> bool {
         given_key_is_coordinator(
             self.signer_public_key(),
             bitcoin_chain_tip,
-            signer_public_keys,
+            &self.context.config().signer.bootstrap_signing_set,
```

Comment:
Using the bootstrap set (unfortunate name we are stuck with I guess) means that when `threshold` signers update the config we will be unable to confirm new requests until we reach the majority of updated configs because of missing coordinator consensus. This seems okay-ish, but I'm wondering if we should instead use the current signer set according to the smart contract -- ie, the last key rotation, together with a fallback for a signer starting fresh without any event.